lib/Backend/GlobOpt.cpp

```diff
@@ -9585,6 +9584,32 @@ GlobOpt::TypeSpecializeInlineBuiltInBinary(IR::Instr **pInstr, Value *src1Val, V
             break;
         }
 
+        case Js::OpCode::InlineMathPow:
+        {
+#ifndef _M_ARM
```

Comment:
This sounds reasonable. I'd like to create a separate work item on this and follow it up after the current one for x86/amd64 because of the difference in details.

---
In reply to: [57610176](https://github.com/Microsoft/ChakraCore/pull/644#discussion_r57610176) [](ancestors = 57610176)